src/figuresClasses.ts

```diff
@@ -1,19 +1,92 @@
+enum Shape {
+  Circle = 'circle',
+  Triangle = 'triangle',
+  Rectangle = 'rectangle',
+}
+
 export interface Figure {
+  color: string,
+  shape: Shape,
+  getArea: () => number,
+}
 
+enum Color {
+  Red = 'red',
+  Green = 'green',
+  Blue = 'blue',
 }
 
-export class Triangle {
+function isValid(...args: number[]): void {
```

Comment:
The name `isValid` refers to the boolean type. Rename function to describe the action it does.